src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -836,19 +837,17 @@ def store_fuzzer_run_results(testcase_file_paths, fuzzer, fuzzer_command,
        fuzzer.return_code == 0 and ' 0/' not in fuzzer.result))
   # pylint: enable=consider-using-in
   if not save_test_results:
-    return
+    return None
 
   logs.log('Started storing results from fuzzer run.')
 
-  # Store the sample testcase in blobstore first. This can take some time, so
-  # do this operation before refreshing fuzzer object.
-  sample_testcase = None
+  fuzzer_run_results_output = uworker_io.StoreFuzzerRunResultsOutput()
   if testcase_file_paths:
     with open(testcase_file_paths[0], 'rb') as sample_testcase_file_handle:
-      sample_testcase = blobs.write_blob(sample_testcase_file_handle)
-
-    if not sample_testcase:
-      logs.log_error('Could not save testcase from fuzzer run.')
+      sample_testcase_file = sample_testcase_file_handle.read()
+      fuzzer_run_results_output.uploaded_sample_testcase = True
+      storage.upload_signed_url(sample_testcase_file,
```

Comment:
It would still be useful to still log in case of upload failure as was done before.

Reply:
This branch was and still is impossible and can never be reached.